bedrock/releasenotes/views.py

```diff
@@ -211,6 +211,9 @@ def releases_index(request, product):
     if product == 'Firefox':
         major_releases = firefox_desktop.firefox_history_major_releases
         minor_releases = firefox_desktop.firefox_history_stability_releases
+        # Firefox 50.1.0 is not an ESR version but should be fallen under 50.0,
+        # so put it in the list as an exception (Bug 1324305).
+        esr_major_versions.append(50)
```

Comment:
Why not just eliminate esr_major_versions entirely and always use the same logic, something along the lines of this: https://github.com/mozilla/bedrock/commit/def6e1239b7b2a0d09550f5fb231afd2961ab953 (untested, I just put this together to clarify what I meant)